config/initializers/simple_form_bootstrap.rb

```diff
@@ -0,0 +1,154 @@
+# Use this setup block to configure all options available in SimpleForm.
+SimpleForm.setup do |config|
```

Comment:
Now that you have SimpleForm installed, go back through the forms in the site and convert them to use the SimpleForm syntax - it's a good way to learn how to use it.